### PR TITLE
Aaron - L242.building_agg

### PR DESCRIPTION
**Pull request by aarony2j**
#599

Didn't pass timeshift test. Went to debug and driver errored out on another module. Please advise. Also, this is my first xml generating chunk, so please review that carefully.
```
[1] "module_aglu_LB125.LC_tot"
Source: local data frame [243 x 6]
Groups: GCAM_region_ID, GLU [243]

   GCAM_region_ID    GLU  year      value change_rate      change
            <int>  <chr> <dbl>      <dbl>       <dbl>       <dbl>
1               2 GLU087  2006 2536.44437   0.9640596 -94.5594601
2               2 GLU100  2006  661.99079   0.9608151 -26.9979347
3               2 GLU151  2006   93.28560   0.9433293  -5.6041541
4               2 GLU170  2006  776.57462   0.9681605 -25.5388609
5               2 GLU171  2006  203.30447   0.9702869  -6.2257884
6               2 GLU177  2006  555.99539   0.9503044 -29.0754437
7               2 GLU179  2006  443.14465   0.9820626  -8.0940355
8               2 GLU183  2006   17.96847   0.9682939  -0.5883638
9               3 GLU080  2006  529.86690   0.9790946 -11.3135694
10              3 GLU086  2006  515.55553   0.9851059  -7.7948333
# ... with 233 more rows
Error in module_aglu_LB125.LC_tot("MAKE", list(L120.LC_bm2_R_UrbanLand_Yh_GLU = list( : 
  ERROR: Interannual fluctuation in global land cover exceeds tolerance threshold of 0.005
```

### CODE REVIEW

File: R/zchunk_batch_building_agg.xml.R [hunk 1]
```diff
@@ -54,9 +52,8 @@ module_energy_batch_building_agg.xml_DISABLED <- function(command, ...) {
     create_xml("building_agg.xml") %>%
       add_xml_data(L242.FinalEnergyKeyword_bld,"FinalEnergyKeyword") %>%
       add_xml_data(L242.SubsectorLogit_bld,"SubsectorLogit") %>%
```
Comment (bpbond) [inline R/zchunk_batch_building_agg.xml.R]: I think you'll need to use `add_logit_tables_xml` instead here.

**Comment by mwisepnnl** [comment]
Aaron and Ben.  I also think this looks well laid out and explained.  Steps are clearly commented and laid out in the r-code.  It looks like the level 2 csv files are reproduced.  Are the xml files reproduced at this point or still need some tweaks?  I haven't reviewed a level 2 yet either so I don't know if I should expect to see the xml files here.

**Comment by aarony2j** [comment]
@bpbond My chunk was similar to Russell's (#665) in that there were if statements where some output tables were not generated. I handled it by deleting the tibble output and corresponding xml line. Russell kept the if statement throughout. Let me know whether you want me to change it.

**Comment by bpbond** [comment]
@aarony2j Well right now the timeshift is bombing with an...unusual...error. Yes, I'd say let's follow @russellhz 's approach and return a tibble with a single NA value. See #665

**Comment by bpbond** [comment]
Nice job @aarony2j - thanks.

**Pull request merged**